### DISCUSSION TITLE
One of the Link to self learning Module is Blocked/Not Opening

### DISCUSSION THREAD

**Discussion by dheeraj26-01** [github/india#773] [Students]
I have completed all the self learning modules except for one, which is of **Azure Security Technologies (AZ-500) : Implement Network Security in Azure.**
The problem is the link wouldn't open, it says request fails every time I open it and when I try opening in a new tab the link says it is **Blocked.**
Please if any of the technicians are seeing this please do the needful, as i have mailed since a week and got no response
<img width="1392" alt="New_Tab Link" src="https://user-images.githubusercontent.com/91659007/153462284-f7b25202-8f00-46a8-9958-e8c3b5e5ec4c.png">
<img width="1392" alt="Request Failed" src="https://user-images.githubusercontent.com/91659007/153462351-3a7d74a0-d526-4cfe-ab13-4c6c66e2fbce.png">
.

**Comment by ey-mentor1** [comment]
Hello,
Please share the screenshots and details mail to FRT support team, they can help you in this issue.
Thank You
